hack/lib/util.sh

```diff
@@ -531,6 +537,17 @@ EOF
 EOF
 }
 
+# list_staging_repos outputs a sorted list of repos in staging/src/k8s.io
+# each entry will just be the $repo portion of staging/src/k8s.io/$repo/...
+# $KUBE_ROOT must be set.
+function kube::util::list_staging_repos() {
+  (
+    cd "${KUBE_ROOT}/staging/src/k8s.io" && \
+    find . -type d -mindepth 1 -maxdepth 1 | cut -c 3- | sort
+  )
```

Comment:
explanation:
- ls output is not portable
- find is more portable but gives relative paths

-> cd to the search directory in a subshell, then trim off the leading `./` from results.
search only for directories, set min/max depth so we only get results immediately below the search dir.